game/src/contrib/systems/DebuggerSystem.java

```diff
@@ -94,13 +94,13 @@ public static void TELEPORT_TO_END() {
     /** Will teleport the Hero on the EndTile so the next level gets loaded */
     public static void LOAD_NEXT_LEVEL() {
         LOGGER.info("TELEPORT ON END");
-        TELEPORT(Game.currentLevel.getEndTile().getCoordinate().toPoint());
+        TELEPORT(Game.endTile().getCoordinate().toPoint());
```

Comment:
@AMatutat Das wäre auch noch ein Punkt: `TELEPORT(Game.endTile().getCoordinate().toPoint())` => `TELEPORT(Game.endTile())`

Hintergrund:
- Warum soll ich als Kunde aus der Koordinate noch einen Punkt machen, also warum kann ich nicht entweder die Koordinate übergeben oder eine Methode in Tile haben, die mir direkt den Punkt liefert?
- Weiter gedacht: Ich habe doch ein Tile, wo ich hin teleportieren will. Warum kann ich das Ding nicht direkt nutzen, also warum muss ich erst nach Koordination oder einem Punkt oder so fingern?

Ist aber wohl kein Thema für diesen PR :)

Reply:
Hab ich eben auch schonmal drüber nachgedacht:

1. Würde ich `Tile` ein `Coordinate coordinate()` geben (für level interne Prozesse) und ein `Point position()` für "externe" Prozesse. Macht es erstmal schöner bis #547 umgesetzt wird
=> Für den User wird Coordinate umrelevanter

> Weiter gedacht: Ich habe doch ein Tile, wo ich hin teleportieren will. Warum kann ich das Ding nicht direkt nutzen, also warum muss ich erst nach Koordination oder einem Punkt oder so fingern?

Ja. Vermutlich ist das nochmal eine Überladung von Methoden wert.

Reply:
Genau - für allgemeine Sachen passen die beiden Methoden (bis #547 da ist), und hier speziell würde es Sinn machen, die Methode direkt für `Tile` zu überladen und zu nutzen. Innen drin kann sich die Methode ja dann die Koordinate oder den Punkt aus dem Tile rausfischen, aber damit habe ich als Nutzer nix mehr zu tun.